test/rhsmlib_test/test_cloud.py

```diff
@@ -142,9 +133,9 @@ def setUp(self):
         """
         Patch communication with metadata provider
         """
-        requests_patcher = patch('rhsmlib.cloud.providers.aws.requests')
-        self.requests_mock = requests_patcher.start()
-        self.addCleanup(requests_patcher.stop)
+        # requests_patcher = patch('rhsmlib.cloud._base_provider.requests')
+        # self.requests_mock = requests_patcher.start()
+        # self.addCleanup(requests_patcher.stop)
```

Comment:
needed / not needed? if not, I'd simply drop it

Reply:
I forgot to remove this commented code.